code/datums/quirks/good.dm

```diff
@@ -53,7 +53,7 @@
 /datum/quirk/empath
 	name = "Empath"
 	desc = "Whether it's a sixth sense or careful study of body language, it only takes you a quick glance at someone to understand how they feel."
-	value = 8
+	value = 4 /// SKYRAT EDIT - Quirk Rebalance - Original: value = 8
```

Comment:
I'd rather this was 6, at least for now. It's better to make small buffs to avoid the risk of overbuffing it. It can always be revisited in the future

Reply:
Personally 4 makes sense for empath, considering it provides very little knowledge, mostly: If they're chaplain, if they're spiritual, mood, and if they have more than 10 tox and oxy damage. But 6 is fair too. Anything lower than 8 works.